tests/test_format.py

```diff
@@ -0,0 +1,84 @@
+"""
+Tests for locidex format module
+"""
+
+import pytest
+from locidex import format
+
+import os
+import json
+from dataclasses import dataclass
+
+EXPECTED_DATA_OUT = "locidex/example/format_db_mlst_out"
+TEST_DATA = "locidex/example/format_db_mlst_in"
+
+@dataclass
+class FormatArgs:
+    input: os.PathLike
+    outdir: os.PathLike
+    min_len_frac: float
+    max_len_frac: float
+    min_ident: float
+    min_match_cov: float
+    translation_table: int
+    not_coding: bool
+    force: bool
+
+@pytest.fixture(scope="module")
+def output_directory(tmp_path_factory):
+    fn = tmp_path_factory.mktemp("build")
+    return str(fn)
+
+
+@pytest.fixture(scope="module")
+def cmd_args(output_directory):
+    command = FormatArgs(
+        input=TEST_DATA,
+        outdir=output_directory,
+        min_len_frac=0.7,
+        max_len_frac=1.3,
+        min_ident=80.0,
+        min_match_cov=80.0,
+        translation_table=11,
+        not_coding=False,
+        force=True
+    )
+    return command
+
+
+def test_format(cmd_args):
+    format.run(cmd_args)
+
+def test_outputs(output_directory):
```

Comment:
Not sure what is the purpose of this function? I suspect it is to make sure the listed files are in the same order. Also on Mac or other OS there might be hidden files such as `.DS_Store` that could break this assertion